tests/slow/DiskFailureCycle.toml

```diff
@@ -3,6 +3,7 @@ buggify = false
 minimumReplication = 3
 minimumRegions = 3
 logAntiQuorum = 0
+storageEngineExcludeTypes = [4]
```

Comment:
Will this eventually be used by RocksDB? If yes, could you please add a TODO?

Reply:
Yes, eventually we should look into ways that would help enable disk chaos on Rocks.